src/stores/store-urls-custom-admin.md

```diff
@@ -60,6 +60,9 @@ As a precaution, do not try to change the Admin URL yourself unless you know how
    {:.bs-callout-info}
    This method allows you to change the Admin Path, but not the Admin URL.
 
+   {:.bs-callout-info}
+   On Magento Cloud, a custom admin path may be set via the **ADMIN_URL** Cloud [variable](https://devdocs.magento.com/cloud/env/environment-vars_magento.html#admin-url) in the Cloud UI. From the technical side, the specified value is added to `app/etc/env.php` within the environment build.
```

Comment:
This is getting a little techie for the user guide. I would keep it brief.

```suggestion
   On Magento Cloud, a custom admin path may be set using the **ADMIN_URL** [variable](https://devdocs.magento.com/cloud/env/environment-vars_magento.html#admin-url) in the Cloud UI.
```

Reply:
I edited this suggestion after the review so that it was clear that the link was taking them to the Cloud guide.

Reply:
Thank you, lady !
I'll implement your suggestion very soon :)